extensions/2.0/Khronos/KHR_animation2/README.md

```diff
@@ -0,0 +1,51 @@
+# KHR\_animation2
+
+## Contributors
+
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)  
+
+## Status
+
+Experimental
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension is based on the animation features of glTF 2.0. The structure of the schemas are the same, however the properties are changed and adapted.  
+  
+The main changes are, that the output value is always a scalar and that the target property is a [JSON Pointer](https://datatracker.ietf.org/doc/html/rfc6901).
+
+Following is summarizing the advantages of this approach:
+
+### Compatibility
+
+Tools e.g. like [Blender](https://www.blender.org/) do follow the same approach:  
+For each channel/property, one can have an individual animation curve.  
+In this apprach, the cases are reduced, where a complete or partial sampling is required.  
+
+### Minimum data set
+
+For e.g. a translation, all three values have to be stored. If the translation is only changed e.g. for the x value, the y and z values always have to be stored as well.  
```

Comment:
When all three components are affected by the animation, there will be three independent channels. The extension should allow interleaving output accessors for better memory management.

Reply:
Hmm, I have to think about this, but I do not see, why this should not be allowed? Isn't the bufferview with the byteStride solving this?

Reply:
The core spec allows buffer views with `byteStride` only for vertex attributes. The extension should simply say that its accessors are allowed to be interleaved as well.